package.json

```diff
@@ -22,20 +24,29 @@
     "test": "jest --projects packages/*",
     "bump": "lerna publish --exact --skip-npm --since \"v$(npm info @appearhere/bloom version)\"",
     "test:travis": "yarn test --coverage --maxWorkers=4",
-    "now-build": "yarn build && yarn build-storybook"
+    "now-build": "yarn build && yarn lint:flow && yarn build-storybook",
```

Comment:
why do we need to lint it on now too?

Reply:
I thought it might be nice to have it there too so they both fail for the same reason

Reply:
I think it'd be useful to have it deployed regardless of flow, since we have at least one failing check